src/mod_authn_file.c

```diff
@@ -600,7 +600,15 @@ static void apr_sha_encode(const char *pw, char *result, size_t nbytes) {
     unsigned char digest[20];
     size_t base64_written;
 
+#ifdef HAVE_WOLFSSL_SSL_H
```

Comment:
see above suggestion for sys-crypto.h

If wolfSSL does not provide SHA1(), then we can create a macro such as
```
#define SHA1(str,len,digest) do { SHA_CTX sha1; SHA1_Init(&sha1); SHA1_Update(&sha1, (const unsigned char *)(str), (len)); SHA1_Final((digest), &sha1); }
```
To avoid potential conflict with user-variable named 'sha1', I would use something a bit more unique, or make this a static inline function.  lighttpd is built to support (most of) C99.

Reply:
Actually, if wolfSSL does not provide SHA1() then it does not need to be an inline function.  It could simply be provided in algo_sha1.[ch].